src/bidiMapper/domains/network/networkRequest.ts

```diff
@@ -153,6 +153,12 @@ export class NetworkRequest {
     );
   }
 
+  delete() {
```

Comment:
let's call it `dispose`?

Reply:
I did it as there was precedent already for delete rather then dispose. OK to change. Should I rename the `BrowsingContextImpl.delete` as well? @thiagowfx

Reply:
Not very opinionated on this. Just please be globally consistent.